### PR TITLE
Scaleout Message Test

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: test/Microsoft.AspNet.SignalR.Server.Tests/ScaleoutMessageFacts.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
```
Comment (moozzyk) [inline test/Microsoft.AspNet.SignalR.Server.Tests/ScaleoutMessageFacts.cs]: Do you need to add a copyright header?

Comment (BrennanConroy) [inline test/Microsoft.AspNet.SignalR.Server.Tests/ScaleoutMessageFacts.cs]: I'm not sure, I looked at a bunch of our other tests and none of them had it.

**Comment by moozzyk** [comment]
:ship::it:

**Comment by BrennanConroy** [comment]
Thanks @moozzyk

**Pull request closed without merging**